sway/config/output.c

```diff
@@ -225,6 +229,10 @@ bool apply_output_config(struct output_config *oc, struct sway_output *output) {
 		sway_log(SWAY_DEBUG, "Set %s scale to %f", oc->name, oc->scale);
 		wlr_output_set_scale(wlr_output, oc->scale);
 	}
+	if (oc && oc->subpixel != WL_OUTPUT_SUBPIXEL_UNKNOWN) {
+		sway_log(SWAY_DEBUG, "Set %s subpixel to %d", oc->name, oc->subpixel);
```

Comment:
If there is a toString function for wl_output_subpixel, I'd love to use it. I looked around for a few minutes but I didn't find one.

Reply:
I think you have to write your own, which should be trivial since it is a small enum

Reply:
I added one to my wlroots PR, since I'm guessing both wlroots and WMs that use it will want to log subpixel order.